ktor-client/ktor-client-core/js/src/io/ktor/client/engine/js/compatibility/Utils.kt

```diff
@@ -22,10 +22,10 @@ internal suspend fun commonFetch(
         controller.abort()
     }
 
-    val promise: Promise<Response> = if (PlatformUtils.IS_NODE) {
-        jsRequire("node-fetch")(input, init)
-    } else {
+    val promise: Promise<Response> = if (PlatformUtils.IS_BROWSER) {
```

Comment:
We saw that sometimes it can work in the opposite. Let's dig in some details: could you tell me the values of the internal predicates in `PlatformUtils.IS_BROWSER` and `IS_NODE` on your platform? Maybe we can force them to work well on each platform.

Reply:
On my platform at the line

```
console.log(window)
// -> global {parent: global, postMessage: f, blur: f, ...}
console.log(window.document)
// -> document { URL: "http://localhost:8088/", activeElement: body, ... }

console.log(process)
// -> process {version: "v12.8.1", versions: {...}, arch: "x64", platform: "darwin", ...}
console.log(process.versions)
// -> {node: "12.8.1", v8: "7.8.279.23-electron.0". uv: "1.30.1", ...}
console.log(process.versions.node)
// -> "12.8.1"
```